frontend/appflowy_flutter/integration_test/desktop/document/document_more_actions_test.dart

```diff
@@ -31,4 +34,37 @@ void main() {
       expect(pageFinder, findsNWidgets(1));
     });
   });
+
+  testWidgets('count title towards word count', (tester) async {
+    await tester.initializeAppFlowy();
+    await tester.tapAnonymousSignInButton();
+    await tester.createNewPageWithNameUnderParent();
+
+    final title = tester.editor.findDocumentTitle('');
+
+    await tester.openMoreViewActions();
+    Finder viewMetaInfo = find.byType(ViewMetaInfo);
+    expect(viewMetaInfo, findsOneWidget);
+
+    ViewMetaInfo viewMetaInfoWidget =
+        viewMetaInfo.evaluate().first.widget as ViewMetaInfo;
+    Counters titleCounter = viewMetaInfoWidget.titleCounters!;
+
+    expect(titleCounter.charCount, 0);
+    expect(titleCounter.wordCount, 0);
+
+    /// input [titleString] within title
+    const titleString = 'Title';
```

Comment:
Add two more operations to detect if the word count has increased. For example:

1. Hello
2. Hello AppFlowy
3. Hello AppFlowy!